src/sfdxProject.ts

```diff
@@ -98,7 +180,13 @@ export class SfdxProject {
    * **Throws** *{@link SfdxError}{ name: 'InvalidProjectWorkspace' }* If the current folder is not located in a workspace.
    */
   public static async resolve(path?: string): Promise<SfdxProject> {
-    return new SfdxProject(await this.resolveProjectPath(path));
+    const _path = path || process.cwd();
```

Comment:
This should be ok (defaulting to `process.cwd()`) since it's done in `resolveProjectPath` anyway.  We need the path for caching.